app/controllers/ui_job_wizard_controller.rb

```diff
@@ -68,9 +68,6 @@ def job_invocation
     job_organization = Taxonomy.find_by(id: job.task.input[:current_organization_id])
     job_location = Taxonomy.find_by(id: job.task.input[:current_location_id])
     render :json => {
-      :provider_input_values => composer[:template_invocations][0][:provider_input_values],
```

Comment:
Don't we need at least one copy of the provider input values? Or are they bundled within the regular inputs?

Reply:
was added here: https://github.com/theforeman/foreman_remote_execution/pull/788/files#diff-494fb439c4dcbef79b8470f33ba943aa649b2d8e698a9bd26c5353e9e33158f7R22

The keys that were deleted were never used in the js side, and js specified which keys it uses.